gradle.properties

```diff
@@ -8,7 +8,7 @@
 # Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
 #
 
-VERSION_NAME=3.7.0-SNAPSHOT
```

Comment:
Why did you update this version? We should release 3.7.0 and only then update this.

Reply:
I'm fine with 3.7.0-SNAPSHOT and then releasing 3.7.0.